docs/assets/environment.md

```diff
@@ -7,18 +7,18 @@ It is possible to set a default value of a package in which `ModelLibrary` will
 | Environment variable | Default value | Notes |
 | --- | --- | --- |
 | `MODELKIT_DEFAULT_PACKAGE` | None | It has to be findable (on the `PYTHONPATH`) |
-| `WORKING_DIR` | None | Local directory to find assets, has to exist |
+| `MODELKIT_ASSETS_DIR` | None | Local directory to find assets, has to exist |
```

Comment:
MODELKIT_ASSETS_LOCAL_DIR ?

Reply:
Since a little bit, the AssetsManager is purely local, and STORAGE would be indicative of the remote store. In addition, there is no notion of a directory on remote stores. 
I think I'd rather keep the `MODELKIT_ASSETS_DIR`, it's long enough already 😬

Reply:
ok for me